test/e2e/lib/driver-helper.js

```diff
@@ -323,7 +323,7 @@ export async function setCheckbox( driver, locator, check = true ) {
  * image is visible or not
  */
 export async function isImageVisible( driver, element ) {
-	const tagName = await element.getTagName();
+	const tagName = ( await element.getTagName() ).toUpperCase();
 	if ( tagName !== 'IMG' ) {
```

Comment:
Nit: calling `toUpperCase()` here requires less chars :)

Reply:
This was fixed here: https://github.com/Automattic/wp-calypso/pull/52832, too; though I couldn't merge yet because it has a fix that is specific to > v10.6. Thanks for adding it here.